src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static final int NUMERIC = 100;
+
+    public static String getRandomBall() {
+
+        Colors color = new ColorSupplier().getRandomColor();
+        int randomCislo = new Random().nextInt(NUMERIC);
```

Comment:
make color supplier and random class fields - now we needlessly recreate these instances per each method call